src/box/box.cc

```diff
@@ -849,6 +849,26 @@ box_check_sql_cache_size(int size)
 	return 0;
 }
 
+static inline bool
+is_exp_of_two(unsigned n)
+{
+	return ( n & (n - 1) ) == 0;
+}
+
+static void
+box_check_small_alloc_options(void)
+{
+	unsigned granularity = cfg_geti("granularity");
```

Comment:
Function returns integer, so if user inputs proper negative value, it'll be accounted as valid.